### PR TITLE
fix: fix the syntax documentation for `watchPosition`

### PR DESCRIPTION
**Pull request by imaxing**
### Description

This pull request fixes a syntax error in the example code for the `watchPosition()` method in the documentation.

### Motivation

The current example code for `watchPosition()` method contains a syntax error in the options parameter. This can be confusing for readers who are trying to learn how to use this method. By fixing this error, we can improve the clarity and accuracy of the documentation.

### Additional details

There are no additional details for this pull request.

### Related issues and pull requests

N/A

### CODE REVIEW

File: files/fr/web/api/geolocation/watchposition/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Cette méthode retourne une valeur ID de veille qui permet de libérer les fonct
 ## Syntaxe
 
 ```js
```
Comment (yin1999) [inline files/fr/web/api/geolocation/watchposition/index.md:22]:
````suggestion
```js-nolint
````

File: files/fr/web/api/geolocation/watchposition/index.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Cette méthode retourne une valeur ID de veille qui permet de libérer les fonct
...
-id = navigator.geolocation.watchPosition(success[, error[, options]])
+id = navigator.geolocation.watchPosition(success, error, options)
```
Comment (yin1999) [inline files/fr/web/api/geolocation/watchposition/index.md:23]:
```suggestion
watchPosition(success)
watchPosition(success, error)
watchPosition(success, error, options)
```

File: files/zh-cn/web/api/geolocation/watchposition/index.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ slug: Web/API/Geolocation/watchPosition
 ## 语法
 
 ```plain
```
Comment (yin1999) [inline files/zh-cn/web/api/geolocation/watchposition/index.md:14]:
````suggestion
```js-nolint
````

File: files/zh-cn/web/api/geolocation/watchposition/index.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ slug: Web/API/Geolocation/watchPosition
...
-id = navigator.geolocation.watchPosition(success[, error[, options]])
+id = navigator.geolocation.watchPosition(success, error, options)
```
Comment (yin1999) [inline files/zh-cn/web/api/geolocation/watchposition/index.md:15]:
```suggestion
watchPosition(success)
watchPosition(success, error)
watchPosition(success, error, options)
```

File: files/zh-tw/web/api/geolocation/watchposition/index.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ slug: Web/API/Geolocation/watchPosition
 ## 語法
 
 ```plain
```
Comment (yin1999) [inline files/zh-tw/web/api/geolocation/watchposition/index.md:14]:
````suggestion
```js-nolint
````

File: files/zh-tw/web/api/geolocation/watchposition/index.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ slug: Web/API/Geolocation/watchPosition
...
-id = navigator.geolocation.watchPosition(success[, error[, options]])
+id = navigator.geolocation.watchPosition(success, error, options)
```
Comment (yin1999) [inline files/zh-tw/web/api/geolocation/watchposition/index.md:15]:
```suggestion
watchPosition(success)
watchPosition(success, error)
watchPosition(success, error, options)
```

**Review by yin1999 — COMMENTED** [review]
Thanks @imaxing. But, could you follow the format in en-US. I've left some suggestions :)

https://github.com/mdn/content/blob/582d8c0192fdc085bf1e33ceb5f4f4acf224f28a/files/en-us/web/api/geolocation/watchposition/index.md?plain=1#L16-L20

**Comment by imaxing** [comment]
@yin1999 Thank you for your suggestions
but I have only compared the document for 'getCurrentPosition' and made revisions. 
I'm not sure if I want to add multiple sample codes :)

**Comment by yin1999** [comment]
> @yin1999 Thank you for your suggestions but I have only compared the document for 'getCurrentPosition' and made revisions. I'm not sure if I want to add multiple sample codes :)

Please, as we are now using the new format for syntax block.

**Comment by imaxing** [comment]
> > @yin1999 Thank you for your suggestions but I have only compared the document for 'getCurrentPosition' and made revisions. I'm not sure if I want to add multiple sample codes :)
> 
> Please use the multiple rows style, as we are now using the new format for syntax block.

No problem, I have made new modifications. If there are any incorrect modifications, please point them out in a timely manner. Thank you:)

File: files/fr/web/api/geolocation/watchposition/index.md [hunk 7]
```diff
@@ -19,10 +19,13 @@ Cette méthode retourne une valeur ID de veille qui permet de libérer les fonct
...
 
-```js
-id = navigator.geolocation.watchPosition(success[, error[, options]])
+```js-nolint
+id = navigator.geolocation.watchPosition(success)
+id = navigator.geolocation.watchPosition(success, error)
+id = navigator.geolocation.watchPosition(success, error, options)
 ```
 
+
```
Comment (yin1999) [inline files/fr/web/api/geolocation/watchposition/index.md:28]: please remove the new line here

File: files/pt-br/web/api/geolocation/watchposition/index.md [hunk 8]
```diff
@@ -11,10 +11,13 @@ Este método retorna um valor para o _watch ID_ que pode ser usado para desregis
...
 ## Síntaxe
 
-```
+```js-nolint
+id = navigator.geolocation.watchPosition(success)
+id = navigator.geolocation.watchPosition(success, error)
 id = navigator.geolocation.watchPosition(success, error, options)
 ```
 
+
```
Comment (yin1999) [inline files/pt-br/web/api/geolocation/watchposition/index.md:20]: here

File: files/zh-tw/web/api/geolocation/watchposition/index.md [hunk 9]
```diff
@@ -11,10 +11,13 @@ slug: Web/API/Geolocation/watchPosition
...
 
-```plain
-id = navigator.geolocation.watchPosition(success[, error[, options]])
+```js-nolint
+id = navigator.geolocation.watchPosition(success)
+id = navigator.geolocation.watchPosition(success, error)
+id = navigator.geolocation.watchPosition(success, error, options)
 ```
 
+
```
Comment (yin1999) [inline files/zh-tw/web/api/geolocation/watchposition/index.md:20]: and here

**Review by yin1999 — APPROVED** [review]
Thanks for the contribution. I've modified those files, as we should use the same format as it in en-US.

The changes now look good to me for l10n-zh.

**Review by SphinxKnight — APPROVED** [review]
ok for @mdn/yari-content-fr

**Comment by SphinxKnight** [comment]
@mdn/yari-content-pt-br this one should be pretty straightforward, let us know what you think :)

**Comment by imaxing** [comment]
> @mdn/yari-content-pt-br this one should be pretty straightforward, let us know what you think :)@mdn/yari-content-pt-br 这个应该非常简单，让我们知道你的想法:)

Um... I'm sorry, I'm not sure what to do next?

**Comment by SphinxKnight** [comment]
> > @mdn/yari-content-pt-br this one should be pretty straightforward, let us know what you think :)@mdn/yari-content-pt-br 这个应该非常简单，让我们知道你的想法:)
> 
> Um... I'm sorry, I'm not sure what to do next?

For now, there is nothing to do on your side :) we are waiting for a review from the `pt-BR` locale team. I'll wait a few days and I'll merge by default.

**Review by josielrocha — APPROVED** [review]
`pt-BR` LGTM!

**Comment by yin1999** [comment]
Thanks. Welcome aboard @imaxing :tada:

**Comment by imaxing** [comment]
> Thanks. Welcome aboard @imaxing 🎉

🤣

**Pull request merged**